src/vcpkg/commands.check-support.cpp

```diff
@@ -147,7 +147,7 @@ namespace vcpkg::Commands
             bool user_supported = false;
 
             std::vector<Port> dependencies_not_supported;
-            for (const auto& action : action_plan.install_actions)
+            for (auto&& action : action_plan.install_actions)
```

Comment:
I'm not sure I agree with this change; the action plan is not obviously 'consumed' here.

Rearranging things such that the action_plan is destroyed after this for loop body would resolve my concern.